enatega-multivendor-admin/src/components/Rider/Rider.jsx

```diff
@@ -166,9 +166,8 @@ function Rider(props) {
         <Box ml={10} mt={1}>
           <label>{t('Available')}</label>
           <Switch
-            defaultChecked={riderAvailable}
-            value={riderAvailable}
-            onChange={e => setRiderAvailable(e.target.checked)}
+            checked={riderAvailable}
+            onChange={() => setRiderAvailable(!riderAvailable)}
```

Comment:
It looks fine but better way to implement this is to use this notation
setRiderAvailable(prevRiderAvailable => !prevRiderAvailable);
It will ensure that the update is based on the most recent state value.